modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/NavigationMenuResourceImpl.java

```diff
@@ -211,6 +222,42 @@ private void _createNavigationMenuItems(
 		}
 	}
 
+	private Class<?> _getClassNameContent(String type, long classPK)
+		throws Exception {
+
+		Object dtoObject = _getDtoObject(type, classPK);
+
+		Class<?> dtoConverterModel = dtoObject.getClass();
```

Comment:
@sergiojimcos all the variable names must end with their type. In this case `dtoConverterModelClass`.